src/sys/unix/selector/epoll.rs

```diff
@@ -168,6 +169,46 @@ pub mod event {
         // Not supported.
         false
     }
+
+    pub fn get_platform_event_data_str(data_str: &mut String, event: &Event) {
+        write!(data_str, "events (").unwrap();
```

Comment:
This is not needed in the `Debug` impl the field name is already added.

Reply:
Removed